ui/admin/app/components/chat/ChatContext.tsx

```diff
@@ -133,6 +142,15 @@ function useMessageSource(threadId?: Nullish<string>) {
             }
 
             if (error) {
+                if (error.includes("thread was aborted, cancelling run")) {
```

Comment:
nit: The backend should probably return some type of discriminator so we don't need to do string parsing to know how to display the UI. Might be worth bringing up to Craig.

That being said it looks like Darren is doing the same thing in the User UI for now, so it's probably fine haha